### DISCUSSION TITLE
Does this support  videos that has 5.1 audio.

### DISCUSSION THREAD

**Discussion by vkkindia** [FreeTubeApp/FreeTube#5345] [Q&A]
Does this support  videos that has 5.1 audio. So that multi channel sounds can be enjoyed. Chrome browser does not support this.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Yes but only in certain case. Havent verified this myself though.

See https://github.com/FreeTubeApp/FreeTube/issues/1581
